bundles/bundle_test.go

```diff
@@ -29,5 +31,177 @@ var _ = Describe("Bundle", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(filepath.Join(dir, "usr", "bin", "edgevpn")).To(BeARegularFile())
 		})
+
+		When("local is true", func() {
+			var installer BundleInstaller
+			var config *BundleConfig
+			var tmpDir, tmpFile string
+			var err error
+
+			BeforeEach(func() {
+				config = &BundleConfig{
+					LocalFile: true,
+				}
+			})
+
+			AfterEach(func() {
+				os.RemoveAll(tmpDir)
+			})
+
+			JustBeforeEach(func() {
+				installer, err = NewBundleInstaller(*config)
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			When("type is container", func() {
+				BeforeEach(func() {
+					tmpDir, err = os.MkdirTemp("", "test")
+					Expect(err).ToNot(HaveOccurred())
+					tmpFile = path.Join(tmpDir, "edgevpn.tar")
+					copyFile("../assets/edgevpn-utils-0.24.4.tar", tmpFile)
```

Comment:
This one is big (32Mb). Let me see if I can find something smaller to add to git.

Reply:
Done (did grub-config intead)